records/validation.go

```diff
@@ -25,31 +25,27 @@ func validateMasters(ms []string) error {
 		}
 		//TODO(jdef) distinguish between intended hostnames and invalid ip addresses
 		if _, found := valid[m]; found {
-			return fmt.Errorf("duplicate master specified: %v", ms[i])
+			return fmt.Errorf("duplicate master specified: %q", ms[i])
 		}
 		valid[m] = struct{}{}
 	}
 	return nil
 }
 
-// validateResolvers checks that each resolver in the list is a properly formatted IP address.
-// duplicate resolvers in the list are not allowed.
-// returns nil if the resolver list is empty, or else all resolvers in the list are valid.
+// validateResolvers errors if there are duplicate resolvers, otherwise returns nil.
```

Comment:
this refactoring is too naive:
- I want duplicate IP's to error
- I want syntactically invalid hostnames to error